submission/proxy_test.go

```diff
@@ -121,3 +122,48 @@ func TestProxyRefreshRootsErr(t *testing.T) {
 		}
 	}
 }
+
+func TestCTPolicyTypeFromString(t *testing.T) {
+	testCases := []struct {
+		name      string
+		from      string
+		wantPlc   CTPolicyType
+		errRegexp *regexp.Regexp
```

Comment:
nit: `wantErr string`? Usually `strings.HasPrefix` is enough to test this kind of thing.